frameworks/cassandra/tests/conftest.py

```diff
@@ -1,12 +1,19 @@
 import pytest
-import sdk_repository as repo
+import sdk_repository
+import sdk_security
 
 
 @pytest.fixture(scope='session')
-def configure_universe():
+def configure_session():
     stub_urls = {}
     try:
-        stub_urls = repo.add_universe_repos()
+        stub_urls = sdk_repository.add_universe_repos()
```

Comment:
Rather than having to repeat this `try: finally:` form in all of the conftest.py files, we could add a few methods
1. A method in sdk_repository:
```
def universe_session():
    stub_urls = {}
    try:
        stub_urls = add_universe_repos()
        yield
    finally:
        remove_universe_repos(stub_urls)
```
2. a method in sdk_security:
```
def service_account_session(account_name: str):
    try:
        setup_security(account_name)
        yield
    finally:
        cleanup_security(account_name)
```
3. Now all the conftest.py files should look like:
```
import pytest
import sdk_repository
import sdk_security

@pytest.fixture(scope='session')
def configure_universe():
    yield from sdk_repository.universe_session()

@pytest.fixture(scope='session')
def configure_session(configure_universe):
    yield from sdk_security.service_account_session('cassandra')
```